bot/cogs/commands/minecraft.py

```diff
@@ -156,23 +156,24 @@ async def mcstatus(self, ctx: Ctx, host=None, port: int = None):
 
             combined = f"{host}{port_str}"
 
-        async with minecraftstatus.MCStatus() as client:
-            try:
-                server = await client.get_server(combined.replace("/", "%2F"))
-                server_card = await client.get_server_card(combined.replace("/", "%2F"))
-
-            except minecraftstatus.errors.ServerNotFound:
-                await ctx.reply(
-                    embed=discord.Embed(
-                        color=self.bot.embed_color,
-                        title=ctx.l.minecraft.mcping.title_offline.format(
-                            self.d.emojis.offline, combined
+        async with SuppressCtxManager(ctx.typing()):
+            async with minecraftstatus.MCStatus() as client:
```

Comment:
You can have context managers on the same line fwiw 👁️ 

`async with x() as y, a() as b:`

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/277/commits/dc60e23cb6a7f5fec506a67b38e7d6f559a24f54